scripts/mock_openai.py

```
@@ -281,14 +283,37 @@ def tc_multi(pairs):
         last_user = next(
             (str(m.get("content", "")) for m in reversed(messages) if m.get("role") == "user"),
             "",
         )
+        if "写授权" in last_user:
+            if n_write > 0:
+                self._respond(text("写授权流程结束。"))
+                return
+            out_dir = os.environ.get("MIQI_AUTH_OUT_DIR", "")
+            target = os.path.join(out_dir, "auth_probe.txt") if out_dir else ""
+            if not target:
+                self._respond(text("写授权：缺少 MIQI_AUTH_OUT_DIR 环境变量。"))
```

Ruff (0.16.2)

[warning] String contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF001)